test/evmc/evmc.h

```diff
@@ -832,12 +832,19 @@ enum evmc_revision
      */
     EVMC_LONDON = 9,
 
+    /**
+     * The Paris revision.
+     *
+     * https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/mainnet-upgrades/paris.md
+     */
+    EVMC_PARIS = 10,
```

Comment:
This is *fine*, but this PR can't be tested without newer evmone, so I'd suggest to take #13532 first.

Reply:
So Paris is the 10th revision? Interesting coincidence.